src/types/Course/Section.ts

```diff
@@ -32,10 +33,124 @@ export const CourseQuery = extendType({
   type: 'Query',
   definition(t) {
     t.list.nonNull.field('courses', {
+      // type is a list of coursesections
       type: CourseSection,
-      description: 'Get a list of courses for a given term',
+      description: 'Get a list of courses for a given term and/or year',
       args: {
         term: arg({ type: Term }),
+        year: arg({ type: 'Int' }),
+      },
+      resolve: async (_, args, ctx) => {
```

Comment:
```suggestion
      resolve: async (_, { term, year }, { prisma }) => {
```
Same as less comment